### ISSUE TITLE
Edit of Existing Issue gives error

### ISSUE THREAD

**Issue by javulorg** [javulorg/javul#68]
Edit of Existing Issue gives error
Steps to reproduce:
1. Login and Visit: http://javul.org/issues/PNGB30RAa8/edit
2. Make any change and try to save. 
3. It will give an error (Column 'file_attachments' cannot be null). Remove this requirement as file attachments is optional.

**Comment by javulorg** [comment]
hi Igor
I'm not able to edit the Issue after its been created e.g.:
http://javul.org/issues/emAnDY12lq/view
If you edit this, and change the description and try to save it wont save.
The only required field for an Issue is the Issue name and the Issue description. 

I notice in the recent code, two more fields are made required, task_id and objective_id.
Those should be removed as they're not required.

So basically we want the ability to create an issue, and just give its name and desc, and then edit it and change those two fields again.
